nengo/utils/tests/test_testing.py

```diff
@@ -72,3 +72,11 @@ def test_logger_norecord():
         logger.info("Testing that logger doesn't record")
     with pytest.raises(ValueError):
         logger_obj.get_filepath(ext='txt')
+
+
+def test_mock_iter(plt):
+    fig = plt.figure()
+    for i, ax in enumerate(fig.axes):
+        if i > 10:
```

Comment:
This line seems somewhat arbitrary and isn't really required because this loop should not even execute once.

Reply:
Feel free to push a fixup